spec/features/user_deletes_review_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+feature "User deletes review" do
+  scenario "User deletes review" do
+
+   artwork = Art.create(
+     name: "Chinatown Community Mural",
+     location: "Surface Rd & Beach St",
+     description: "Chinese community mural commissioned
+     by the mayor's mural crew.",
+     artist: "Made up Artist",
+     category: "Graffiti"
+   )
```

Comment:
I would recommend using a factory for creating your artworks - this is a really big chunk of code just for creating one object in the database!